src/common/test-webserver.c

```diff
@@ -1113,5 +1135,38 @@ main (int argc,
   g_test_add ("/web-server/for-tls-proxy", TestFixture, &test_case_for_tls_proxy,
               setup, test_webserver_for_tls_proxy, teardown);
 
+  /* Header is enabled, but not passed.  Default to "http" */
+  static const TestCase test_case_proto_header_empty = {
```

Comment:
I find it quite nice to declare the TestCase right here; but for consistency, please move the existing global ones as well (in a preceeding commit), or move the new ones to the global scope as well.

Reply:
This seems like an awful lot of churn...

Reply:
I did some macro crack to make this feel a lot nicer, and changed all cases (at least in this one file) over to use the new stuff.